### ISSUE TITLE
Steel and iron pipes should have investment costs (they are zero now)

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#1330]
[https://github.com/quintel/etmoses/blob/master/db/static/gas_assets/pipes/125_mbar/steel_lp.yml](https://github.com/quintel/etmoses/blob/master/db/static/gas_assets/pipes/125_mbar/steel_lp.yml)

**Comment by Sanne512** [comment]
Steel is not used in the low pressure range any more, only the high pressure range. So there are no investments being made any more in low pressure steel pipes, so there is also no indication available of these investment costs. The reason to include them in the oversight is because there are still steel pipes present in the ground. Is this explanation clear?

**Comment by ChaelKruip** [comment]
> So there are no investments being made any more in low pressure steel pipes, 

Yes, but their deprecation is still ongoing in some cases right? To take those costs into account we need to know what the original investment costs was.

> so there is also no indication available of these investment costs.

That is unfortunate. Are these numbers known from earlier years when they were still actively placed?

Or, alternatively, can we use the investment costs of a similar type of pipe?

**Comment by ChaelKruip** [comment]
If we cannot find any costs, I think we should disable this type of pipe because now the user can build infrastructure for free.

**Comment by Sanne512** [comment]
> their deprecation is still ongoing in some cases right

Yes

> Are these numbers known from earlier years when they were still actively placed?

That would give a wrong indication of the costs, due to the way the costs are calculated: the costs of the other pipes exist of aggregated costs of all the pipes of one material aggregated at one pressure level. (With aggregrated at one pressure level I mean for example that 125 mbar represents the pressure levels between 100 and 200 mbar.) This means that the number for the costs includes a wide array of diameters,  a wide array of digging costs in different areas, etcetera. To ensure that the costs for all the different items of the gas infrastructure are calculated in a similar method it wouldn't be "fair" to use the numbers for the steel pipes from earlier years when these were still actively placed. 

> can we use the investment costs of a similar type of pipe?

@MarjoleinSchwachofer : could you check with one of the gas experts if the costs of the PVC or PE pipes at a low pressure level are comparable? 

> If we cannot find any costs, I think we should disable this type of pipe because now the user can build infrastructure for free.

I agree. 

I'm also including @ThomBuijs in this discussion.

**Comment by ChaelKruip** [comment]
Assigned @grdw to remove the pipes with zero investment costs (are there more than one?)

**Comment by grdw** [comment]
There are more than one. 

It are these pipes:
````
pipes/125_mbar/steel_lp.yml
pipes/4_bar/ductile_iron_hp.yml
pipes/8_bar/ductile_iron_hp.yml
````

**Comment by MarjoleinSchwachofer** [comment]
We will check if we can get the values from another source

**Comment by ChaelKruip** [comment]
> We will check if we can get the values from another source

👍 

Removing them for now. We can revert #1369 if @MarjoleinSchwachofer can find values.

**Comment by ChaelKruip** [comment]
Re-opened to keep this on the radar.
